jobs/uaa/spec

```diff
@@ -451,6 +451,9 @@ properties:
   login.checkOriginEnabled:
     description: "This flag enables the origin check in SCIM. Otherwise, the assignments of users to an origin are not validated."
     default: false
+  login.allowOriginLoop:
+    description: "This flag enables the loop over all origin of a certain type during login, e.g. all SAML or OIDC providers in case of such a logon. Otherwise, only index access is allowed."
```

Comment:
```suggestion
    description: "This flag enables the loop over all origins of a certain type during login, e.g. all SAML or OIDC providers in case of such a logon. Otherwise, only index access is allowed."
```